src/services/MeasureService.ts

```diff
@@ -88,4 +89,31 @@ export class MeasureService implements Service<fhir4.Measure> {
 
     return createMeasurePackageBundle(measure[0]);
   }
+
+  /**
+   * result of sending a POST or GET request to:
+   * {BASE_URL}/4_0_1/Measure/$data-requirements or {BASE_URL}/4_0_1/Measure/:id/$data-requirements
+   * creates a Library with all data requirements for the specified measure
+   * requires parameters id and/or url and/or identifier, but also supports version as supplemental (optional)
+   */
+  async dataRequirements(args: RequestArgs, { req }: RequestCtx) {
+    logger.info(`${req.method} ${req.path}`);
+    logger.info('Using package to create measure bundle');
+    const path = req.path.replace(/data-requirements/i, 'package');
+    const measureBundle = await this.package(args, { req: { ...req, path: path } });
+
+    // TODO: Clarify should period start/end be required or have intelligent defaults in fqm-execution to measure effectivePeriod? (fqm-execution defaults to 2019)
+    // short term: default to 2022 if these aren't passed through
```

Comment:
My last question (promise!) is whether we should change this TODO based on what Matt learned at the connectathon.

Reply:
Matt's [connectathon branch](https://github.com/projecttacoma/measure-repository-service/commit/615f33a5fa635b5301ebeb47436c91138557ab29) makes the update and removes this TODO, so I think it's fine to leave it here since the actual changes will come through in his PR (meaning this will be in the main branch for a v short period of time)